httpd/handler.go

```diff
@@ -248,7 +248,7 @@ func (h *Handler) serveStatus(w http.ResponseWriter, r *http.Request) {
 	} else {
 		b, _ = json.Marshal(data)
 	}
-	w.Write(b)
+	_, _ = w.Write(b)
```

Comment:
Should we be using ignore for some of this;

`errcheck -ignore '[rR]ead|[wW]rite' path/to/package`

I feel like this creates a lot of noise for no reason in a package like this.

I agree with the errcheck concept, but there is good reason you don't need to check some of these.